packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -52,6 +53,14 @@ const Accordion = ({
   animationTiming,
   ...props
 }: AccordionProps) => {
+  const elementId = useId();
+  const reducedMotionQuery = '(prefers-reduced-motion: reduce)';
+  const reducedMotionEnabled = useMediaQuery(reducedMotionQuery, true);
+
+  if (reducedMotionEnabled) {
+    animationTiming = false;
+  }
```

Comment:
If window (e.g., apps using SSR) is not available or window.matchMedia is otherwise undefined (e.g., in tests), this will err on the side of caution and assume the user has `prefer-reduce-motion: reduce` enabled. 

I recommend defaulting to true, as the impact of animations being mistakenly disabled is less of an issue than the alternative.